### DISCUSSION TITLE
Auto refresh table with records

### DISCUSSION THREAD

**Discussion by amigolj** [Laravel-Backpack/community-forum#1167] [Q&A (Help)] [answered]
Hi, could you please help me how can I implement the auto-refresh table if in the db  added new records

**Comment by jcastroa87** [comment, marked as answer]
Hello @amigolj 

I dont think we have this featured like a reactive refresh, but with datatables (https://datatables.net/) there is a method on window.crud.table that you can you use; ajax.reload(), just wrap it inside a setInternal

`setInterval(window.crud.table.ajax.reload, 5000)`

Cheers.

**Reply by amigolj** [reply]
Thanks

**Comment by jcastroa87** [comment]
Due to not activity, i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
